src/LaunchDarkly.TestHelpers/HttpTest/SimpleJsonService.cs

```diff
@@ -210,35 +230,44 @@ private void AddPathInternal(HttpMethod method, string path, Handler handler)
                 var result = await handler(context);
                 if (typeof(TOutput).IsValueType || !result.Body.Equals(default(TOutput)))
                 {
-                    await Handlers.BodyJson(JsonConvert.SerializeObject(result.Body))(context);
+                    await Handlers.BodyJson(JsonConvert.SerializeObject(result.Body, _jsonSerializerSettings))(context);
                 }
                 return result.Base;
             });
 
         private Handler Wrap<TInput, TOutput>(Func<IRequestContext, TInput, Task<SimpleResponse<TOutput>>> handler) =>
             Wrap(async context =>
             {
-                if (!ParseInput<TInput>(context, out var input))
-                {
-                    return SimpleResponse.Of(400);
-                }
+                var input = ParseInput<TInput>(context);
                 var result = await handler(context, input);
-                await Handlers.BodyJson(JsonConvert.SerializeObject(result.Body))(context);
+                await Handlers.BodyJson(JsonConvert.SerializeObject(result.Body, _jsonSerializerSettings))(context);
                 return result.Base;
             });
 
-        private bool ParseInput<TInput>(IRequestContext context, out TInput result)
+        private TInput ParseInput<TInput>(IRequestContext context)
         {
             try
             {
-                result = JsonConvert.DeserializeObject<TInput>(context.RequestInfo.Body);
-                return true;
+                return JsonConvert.DeserializeObject<TInput>(context.RequestInfo.Body, _jsonSerializerSettings);
             }
-            catch (JsonException)
+            catch (JsonException e)
             {
-                result = default(TInput);
-                return false;
+                throw new BadRequestException(e.Message);
             }
         }
+
+        private static JsonSerializerSettings MakeJsonSerializerSettings(IList<JsonConverter> converters)
+        {
+            return new JsonSerializerSettings
+            {
+                ContractResolver = new CamelCasePropertyNamesContractResolver(),
```

Comment:
Also, specifying `CamelCasePropertyNamesContractResolver` makes it so the default serialization behavior removes the initial capitalization from .NET property names. That's pretty much always what we want, but it can always be overriden with an attribute on a particular property.